refinery/core/views.py

```diff
@@ -1109,11 +1102,44 @@ def get_user(self, id):
             raise APIException("Multiple users returned for this request.")
 
     def post(self, request, uuid):
-        # if group is a manager_group, user is promoted by adding to user set
+        ''' handles promoting, demoting, and removing users. Note, adding
+        members to a group handled to Invitations API.'''
```

Comment:
Updates to an existing resource should be made via a PUT or a PATCH (https://restfulapi.net/rest-put-vs-post/).